deid/data/rsna-anon-2022.dicom

```diff
@@ -0,0 +1,4127 @@
+FORMAT dicom
+
+%filter whitelist
+
+LABEL CT Siemens Sensation 64 Catch-all # (Susan Weber)
```

Comment:
This looks like it includes the original content from dicom.deid - could we have the rsna file with just entries from RSNA? other than helping the review, I think if someone wanted both the default in deid and this one they could load them both. But if they just wanted the RSNA you can't easily separate / identify it here. What do you think?

Reply:
Yes I just appended all the extra directives to dicom.deid - great idea re separating them. I can generate just the directives from the RSNA protocol and update the PR. 

For my own interest what's the easiest way to combine two recipes? 

Also I think order may matter in the file, for example in dicom.deid there is
```
REMOVE PatientID
...
ADD PatientID var:entity_id
```

naively combining the two recipes might have intentional consequences if say recipe 2 had another `REMOVE PatientID`

Reply:
see "Combine Recipes" in this section: https://pydicom.github.io/deid/examples/recipe/#combine-recipes. We don't have a command line way to do this (yet) so if that is something you'd be interested in, we could add! We would essentially make the `--deid` parameter an append to allow more than one.

> naively combining the two recipes might have intentional consequences if say recipe 2 had another REMOVE PatientID

indeed it would - the original library tried to provide dicom.deid as a single "best effort" recipe to handle most cases, but if this causes trouble we might want to consider either separating things, or allowing to load a recipe without using some section.